src/main/java/sorinfoca/driver/BrowserManager.java

```diff
@@ -7,15 +7,29 @@
 
 import java.net.MalformedURLException;
 import java.net.URL;
+
 import java.util.HashMap;
 import java.util.Map;
 
+
 public class BrowserManager {
     public static ChromeDriver createChromeDriver() {
         WebDriverManager.chromedriver().setup();
         return new ChromeDriver();
     }
 
+
+    RemoteWebDriver driver;
```

Comment:
Please remove these lines from 21-32, no need to define it twice. You already have defined the method for the RemoteWebDriver below -> 
![image](https://user-images.githubusercontent.com/18215854/215026551-56c34b93-2265-4045-9dc5-86b04620341a.png)

Reply:
I don't see anything like that in my code
![Untitlszsafasfed](https://user-images.githubusercontent.com/117103426/215079738-1afdf577-5fc0-4496-8c83-86ca993c3c44.jpg)